shared/login/recover-password/reset-password/index.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import {SignupScreen, InfoIcon} from '../../../signup/common'
+import {ButtonType} from '../../../common-adapters/button'
+
+export type Props = {
+  onContinue: (choice: boolean) => void
+}
+
+const ResetPassword = (props: Props) => {
+  return (
+    <SignupScreen
+      buttons={[
+        {
+          label: 'Reset password',
```

Comment:
"Send reset link"? I had no idea what this was doing till i got the email. cc @cecileboucheron